src/Instrumentation/Symfony/tests/Integration/MessengerInstrumentationTest.php

```diff
@@ -35,14 +35,26 @@ protected function getMessenger(): MessageBusInterface
     {
         return new MessageBus();
     }
+
     protected function getTransport()
     {
-        // Symfony 6+
+        // Symfony 6+ version of the transport
         if (class_exists(InMemoryTransport::class)) {
             return new InMemoryTransport();
         }
 
-        // Symfony 5+
+        // Symfony 5+ fallback
+        return new LegacyInMemoryTransport();
+    }
+
+    protected function getReceiver()
```

Comment:
is this different enough from `getTransport` to require a method? It doesn't return a receiver but a transport...

Reply:
@RichardChukwu have you had any time to look further into this?

Reply:
Hey @ChrisLightfootWild not really, I'll appreciate any assistance please